InvenTree/company/models.py

```diff
@@ -423,12 +423,15 @@ def pretty_name(self):
         return str(self)
 
     def __str__(self):
-        s = "{supplier} ({sku})".format(
-            sku=self.SKU,
-            supplier=self.supplier.name)
+        s = ''
+
+        if self.part.IPN:
+            s += f'{self.part.IPN}'
+
+        s += f' | {self.supplier.name} | {self.SKU}'
```

Comment:
In the case where the part does *not* have an IPN, this will result in a leading space and pipe character.

I think the following approach would be cleaner:

```python
if self.part.IPN:
    s += f'{self.part.IPN}'
    s += ' | '

s += f'{self.supplier.name} | {self.SKU}`
```

Reply:
Yes right, much better indeed! Just updated the PR! :zap: